pwned_passwords_django/api.py

```diff
@@ -13,10 +15,19 @@ def pwned_password(password):
     """
     password_hash = hashlib.sha1(password.encode('utf-8')).hexdigest().upper()
     prefix, suffix = password_hash[:5], password_hash[5:]
-    results = [
-        int(line.split(':')[1]) for line in
-        requests.get(API_ENDPOINT.format(prefix)).text.splitlines()
-        if line.startswith(suffix)
-    ]
+    try:
+        results = [
+            int(line.partition(':')[2])
```

Comment:
Curious why you prefer `partition` here. Is it just to avoid also trying to catch `IndexError`?